Cargo.toml

```diff
@@ -56,6 +56,8 @@ fastdivide = "0.3"
 itertools = "0.10.0"
 measure_time = "0.8.0"
 
+nuclia_vectors = { git = "https://github.com/nuclia/nuclia_vectors.git" }
```

Comment:
Will this be behind a feature?

Reply:
I'm not sure. I can see that if the vector indexation is not well integrated into the Segments readers and writers, and it's like it's calling an external library, it could be behind a feature.

Although, if the code of this PR is well integrated (and there is still work to do :-) ), there is no need to make a feature to support Vector type fields. This will be only one more dependency.